src/js/components/Booking.js

```diff
@@ -97,7 +216,51 @@ class Booking {
     thisBooking.dom.wrapper.addEventListener('updated', function () {
       thisBooking.updateDOM();
     });
+    thisBooking.dom.tablesContainer.addEventListener('click', function (event) {
+      thisBooking.initTables(event.target);
+    });
+    thisBooking.dom.form.addEventListener('submit', function (event) {
```

Comment:
I tutaj powinnaś odwołać się do `thisBooking.dom.submit
```suggestion
    thisBooking.dom.submit.addEventListener('submit', function (event) {
  ```